jetson/percep/pcl.hpp

```diff
@@ -7,19 +7,29 @@
 /* --- Compare Line Class --- */
 //Functor that indicates where a point is in
 //relation to a line in 2D space
+//recieves angle off y axis
```

Comment:
See previous comment on x axis. Please explicitly define the y axis in your context for clarity.